src/main/java/org/kainos/ea/controller/JobController.java

```diff
@@ -0,0 +1,39 @@
+package org.kainos.ea.controller;
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.JobService;
+import org.kainos.ea.client.CantGetAnyRolesException;
+import org.kainos.ea.db.DatabaseConnector;
+import org.kainos.ea.db.JobDao;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("API for Jobs Controller")
+@Path("/api")
+public class JobController {
+
+  private JobService jobService;
+
+  public JobController(){
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+    jobService = new JobService(new JobDao(), databaseConnector);
```

Comment:
Instead of using `new JobDao()` you can create a class variable instead in case you need to use `JobDao` again. Or alternatively create the `JobDao` inside the `constructor`